Common/src/CConfig.cpp

```diff
@@ -1262,8 +1239,8 @@ void CConfig::SetConfig_Options() {
   /*!\brief INC_DENSITY_INIT \n DESCRIPTION: Initial density for incompressible flows (1.2886 kg/m^3 by default) \ingroup Config*/
   addDoubleOption("INC_DENSITY_INIT", Inc_Density_Init, 1.2886);
   /*!\brief INC_VELOCITY_INIT \n DESCRIPTION: Initial velocity for incompressible flows (1.0,0,0 m/s by default) \ingroup Config*/
-  default_vel_inf[0] = 1.0; default_vel_inf[1] = 0.0; default_vel_inf[2] = 0.0;
-  addDoubleArrayOption("INC_VELOCITY_INIT", 3, Inc_Velocity_Init, default_vel_inf);
+  vel_init[0] = 1.0; vel_init[1] = 0.0; vel_init[2] = 0.0;
+  addDoubleArrayOption("INC_VELOCITY_INIT", 3, vel_init);
```

Comment:
@pcarruscag Ok I see. This really seems so straight forward with what it replaces but it isn't